### PR TITLE
(EJ2-845677): Added Volume 3 features samples in TS and JS UG.

### PR DESCRIPTION
**Pull request by Soundeswari-Selvam**
**Description:**

Added Volume 3 features sample.

### CODE REVIEW

File: ej2-javascript/chart/chart-print.md [hunk 1]
```diff
@@ -114,4 +114,97 @@ Initially, render more than one chart to export, and then add button to export t
 {% endtabs %}
 
 {% previewsample "page.domainurl/code-snippet/chart/export-cs3" %}
+{% endif %}
+
+## Adding header and footer in PDF export
+
+You can customize text, page number, line, page size and changing orientation in header and footer.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-print.md:121]: The PDF export option allows you to include a header and footer in the exported chart, where you can add text.

File: ej2-javascript/chart/chart-print.md [hunk 2]
```diff
@@ -114,4 +114,97 @@ Initially, render more than one chart to export, and then add button to export t
...
+{% include code-snippet/chart/export-cs5/index.html %}
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/chart/export-cs5" %}
+{% endif %}
+
+## Exporting charts into multiple page
+
+You can export the multiple charts in multiple page by passing the multiple chart objects in the export method of chart.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-print.md:183]: You can export charts from the web page onto multiple pages within a PDF document.

File: ej2-javascript/chart/synchronized-chart.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+domainurl: ##DomainURL##
+---
+
+# Synchronized Charts in ##Platform_Name## Chart control
+
+## Synchronized charts
+
+This sample showcases the synchronization of multiple charts, with crosshair, tooltip, and zooming functionalities synchronized across the charts. Hover over or zoom in on one chart to observe the corresponding impact on the other charts as well.
+
+{% if page.publishingplatform == "typescript" %}
```
Comment (Gowrimathi) [inline ej2-javascript/chart/synchronized-chart.md:18]: This is not all what we discussed. We need to explain about how to synchronize tooltip, crosshair, zooming and selection.

**Review by Gowrimathi — CHANGES_REQUESTED** [review]
Requested changes

File: ej2-javascript/chart/chart-print.md [hunk 4]
```diff
@@ -114,4 +114,97 @@ Initially, render more than one chart to export, and then add button to export t
 {% endtabs %}
 
 {% previewsample "page.domainurl/code-snippet/chart/export-cs3" %}
+{% endif %}
+
+## Adding header and footer in PDF export
+
+The PDF export option allows you to include a header and footer in the exported chart, where you can add text.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-print.md:121]: In the export method, specify the following parameters to add a header and footer to the exported PDF document:

File: ej2-javascript/chart/chart-print.md [hunk 5]
```diff
@@ -114,4 +114,97 @@ Initially, render more than one chart to export, and then add button to export t
...
+{% include code-snippet/chart/export-cs4/index.html %}
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/chart/export-cs4" %}
+{% endif %}
+
+## Excel export
+
+The rendered chart can be exported to `XLSX` or `CSV` format using the excel export method in chart. The input parameters for this method are `Export Type` for format and `fileName` for result.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-print.md:152]: You can export the rendered chart data to XLSX or CSV formats using the export method.

In the export method, you'll need to specify the following parameters in order to export the data to Excel:

File: ej2-javascript/chart/chart-print.md [hunk 6]
```diff
@@ -114,4 +114,97 @@ Initially, render more than one chart to export, and then add button to export t
...
+{% include code-snippet/chart/export-cs5/index.html %}
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/chart/export-cs5" %}
+{% endif %}
+
+## Exporting charts into multiple page
+
+You can export charts from the web page onto multiple pages within a PDF document.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-print.md:183]: In the export method, specify the following parameter to export charts to multiple pages in a PDF format.

**Review by Gowrimathi — CHANGES_REQUESTED** [review]
Requested changes

File: ej2-javascript/chart/synchronized-chart.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+publishingplatform: ##Platform_Name##
+documentation: ug
+domainurl: ##DomainURL##
+---
+
+# Synchronized Charts in ##Platform_Name## Chart control
+
+## Tooltip synchronization
+
+Tooltip synchronization enhances the data visualization experience by seamlessly coordinating tooltips across multiple charts. When we hover over a data point in one chart, synchronized tooltips intelligently display related information in other connected charts simultaneously.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/synchronized-chart.md:16]: Synchronize the tooltips across the multiple charts using 'showTooltip' and 'hideTooltip' methods. When we hover over a data point in one chart, call the showTooltip method for the rest of the chart to display related information in other connected charts simultaneously.

In the showTooltip method, specify the following parameter programmatically to enable tooltips for a particular chart:

'x' - Data point x-value or x-coordinate value.
'y' - Data point y-value or y-coordinate value.

File: ej2-javascript/chart/synchronized-chart.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+{% include code-snippet/chart/user-interaction-cs45/index.html %}
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/chart/user-interaction-cs45" %}
+{% endif %}
+
+## Crosshair synchronization
+
+Crosshair synchronization ensures that as we navigate one chart, the crosshair cursor precisely aligns with data points in other linked charts, simplifying data comparison and analysis.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/synchronized-chart.md:47]: Synchronize the crosshair across multiple charts using the 'showCrosshair' and 'hideCrosshair' methods. When hovering over one chart, call the 'showCrosshair' method for the rest of the charts to align with data points in other linked charts, simplifying data comparison and analysis.

In the 'showCrosshair' method, specify the following parameter programmatically to enable crosshairs for a particular chart:

'x' - Specifies the x value of the point or x-coordinate.
'y' - Specifies the y value of the point or y-coordinate.

**Review by Gowrimathi — CHANGES_REQUESTED** [review]
Requested added

File: ej2-javascript/chart/synchronized-chart.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+{% include code-snippet/chart/user-interaction-cs46/index.html %}
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/chart/user-interaction-cs46" %}
+{% endif %}
+
+## Zooming synchronization
+
+Zoom synchronization ensures coherence across multiple charts by maintaining consistent zoom levels. It delivers a unified perspective of  data, facilitating comprehensive analysis at various levels of granularity.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/synchronized-chart.md:78]: You can maintain consistent zoom levels across multiple charts using the `zoomComplete` event. In the zoomComplete event, obtain the `zoomFactor` and `zoomPosition` values of the particular chart and then apply the same values to the rest of the charts.

File: ej2-javascript/chart/synchronized-chart.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+{% include code-snippet/chart/user-interaction-cs47/index.html %}
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/chart/user-interaction-cs47" %}
+{% endif %}
+
+## Selection synchronization
+
+Selection synchronization streamlines data analysis by highlighting related data points in all connected charts when we make a selection, enabling seamless comparison.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/synchronized-chart.md:109]: You can select the data across multiple charts using the `selectionComplete` event. In the selectionComplete event, obtain the selected values of the current chart and then apply the same values to the rest of the charts.

**Review by Gowrimathi — CHANGES_REQUESTED** [review]
Requested changes

File: ej2-javascript/chart/chart-print.md [hunk 11]
```diff
@@ -114,4 +114,107 @@ Initially, render more than one chart to export, and then add button to export t
 {% endtabs %}
 
 {% previewsample "page.domainurl/code-snippet/chart/export-cs3" %}
+{% endif %}
+
+## Adding header and footer in PDF export
+
+In the export method, specify the following parameters to add a header and footer to the exported PDF document:
+
+* `header` - In the header, we can specify the text that is to appear at the top of the exported PDF.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-print.md:123]: Specify the text that should appear at the top of the exported PDF.
Specify the text that should appear at the Bottom of the exported PDF.

File: ej2-javascript/chart/chart-print.md [hunk 12]
```diff
@@ -114,4 +114,107 @@ Initially, render more than one chart to export, and then add button to export t
...
+{% endif %}
+
+## Excel export
+
+You can export the rendered chart data to `XLSX` or `CSV` formats using the export method.
+
+In the export method, you'll need to specify the following parameters in order to export the data to Excel:
+
+* `type` - You can specify a type, which can be either `XLSX` or `CSV` format, for exporting in Excel.
+* `fileName` - You can specify a name for the exported documents.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-print.md:160]: Specify the export format as either 'XLSX' or 'CSV' for exporting in Excel.
A name for the exported file.

File: ej2-javascript/chart/chart-print.md [hunk 13]
```diff
@@ -114,4 +114,107 @@ Initially, render more than one chart to export, and then add button to export t
...
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/chart/export-cs5" %}
+{% endif %}
+
+## Exporting charts into multiple page
+
+In the export method, specify the following parameter to export charts to multiple pages in a PDF format.
+
+* `exportToMultiplePage` - You can enable `exportToMultiplePage` to export each chart on an individual page.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-print.md:193]: Set 'exportToMultiplePage' to true in order to export each chart on an individual page.

**Review by Gowrimathi — CHANGES_REQUESTED** [review]
Requested changes

File: ej2-javascript/chart/chart-print.md [hunk 14]
```diff
@@ -79,19 +79,80 @@ The optional parameters for this method are,
...
+{% include code-snippet/chart/export-cs4/index.html %}
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/chart/export-cs4" %}
+{% endif %}
+
+### Exporting charts into separate page during the PDF export
+
+During the PDF export, you can export each chart as a separate page by setting the `exportToMultiplePage` parameter to **true**.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-print.md:118]: During PDF export, set the exportToMultiplePage parameter to true to export each chart as a separate page.

**Review by Gowrimathi — APPROVED** [review]
Fine

**Review by Gowrimathi — APPROVED** [review]
Fine

**Comment by Soundeswari-Selvam** [comment]
rebuild

**Pull request merged**